ShippingZones.tsx

```diff
@@ -93,6 +95,13 @@ const ShippingZonesList = () => {
 
   useEffect(() => {
     navigation.setOptions({
+      headerLeft: () => <HeaderBackButton
+          style={{marginLeft: Platform.OS === "ios" ? -15 : -5}}
+          labelVisible={Platform.OS === "ios"}
+          label="Settings"
+          onPress={() => {
+            NativeModules.ExitModule.exit();
+          }}/>,
```

Comment:
I've introduced [`HeaderBackButton`](https://reactnavigation.org/docs/elements/#headerbackbutton) to have back navigation buttons to look native on both Android and iOS.
`
But, as we use [`Native Stack Navigator`](https://reactnavigation.org/docs/native-stack-navigator), the `HeaderBackButton`, which is default control for the [`Stack Navigator`](https://reactnavigation.org/docs/stack-navigator) is a little off - that's why I needed to set custom margins etc.

After reading more about Native Stack Navigator and Stack Navigator, I think we should consider using former. I've added issue to track that #55 but it's not high priority I believe